ecc/enclave/enclave_stub.go

```diff
@@ -255,7 +257,7 @@ type Stub interface {
 	// Return report and enclave PK in DER-encoded PKIX format
 	GetLocalAttestationReport(targetInfo []byte) ([]byte, []byte, error)
 	// Invoke chaincode
-	Invoke(args []byte, pk []byte, shimStub shim.ChaincodeStubInterface, tlccStub tlcc.TLCCStub) ([]byte, []byte, error)
+	Invoke(args []byte, shimStub shim.ChaincodeStubInterface) ([]byte, error)
 	// Returns enclave PK in DER-encoded PKIX formatk
 	GetPublicKey() ([]byte, error)
```

Comment:
This is obsolete? Don't have to necessarily purge it here but would seem good to do so sooner than later as this can be confusing. Actually, here it probably will be automatically purged by marcus' pr but it wouldn't inside `ecc_enclave`?
```suggestion
        // TODO: purge this obsolete function here and in ecc_enclave.
	// Returns enclave PK in DER-encoded PKIX formatk
	GetPublicKey() ([]byte, error)
```

Reply:
done